settings.gradle

```diff
@@ -26,4 +26,5 @@ include ':autodispose-kotlin'
 include 'autodispose-rxlifecycle'
 include ':sample'
 include ':test-utils'
+include ':tooling:autodispose-error-prone-checker'
```

Comment:
Let's rename `tooling` to `static-analysis`. I suspect you borrowed the `tooling` convention from the internal monorepo, but there's not actually any build tooling here

Reply:
Also changing the name in gradle.properties for the artifact.

Reply:
No the artifact name is fine. Only the directory needs to change

Reply:
https://github.com/uber/AutoDispose/pull/156/commits/5de25a52604028f03ad5590652e0f79997a8f443.